README.md

```diff
@@ -134,45 +134,20 @@ for an org registered under `acme.radicle.eth`. Replace `acme` with the name
 of your org.
 
 ### Docker
+#### As an end user
 
-There are `Dockerfile` provided for both services in the respective directories.
+ * Make sure the *identity* file can be found under `$HOME/.radicle/identity`.
+ * Replace your org address with your own in `docker-compose.yml`, in the argument to the `--orgs` option.
+ * Don't forget to specify the `--bootstrap` option if needed.
 
-#### Building
+1. [Install docker-compose](https://docs.docker.com/compose/install/).
+1. `docker-compose pull`
+1. `docker-compose up --detach`
 
-To build the containers, run:
+#### As a developer, building from source and pushing for deployment
 
-    $ docker build -t radicle-services/org-node -f org-node/Dockerfile .
-    $ docker build -t radicle-services/http-api -f http-api/Dockerfile .
-
-#### Running
-
-To run the org node after it is built, you can run for example:
-
-    $ docker run \
-        --init \
-        -e RUST_LOG=info \
-        -p 8776:8776 \
-        -v $HOME/.radicle:/app/radicle radicle-services/org-node \
-        --subgraph https://api.thegraph.com/subgraphs/name/radicle-dev/radicle-orgs \
-        --orgs 0xceAa01bd5A428d2910C82BBEfE1Bc7a8Cc6207D9 \
-        --rpc-url ws://localhost:8545
-
-Make sure the *identity* file can be found under `$HOME/.radicle/identity` and
-that you replace the org address with your own. Don't forget to specify the
-`--bootstrap` option if needed.
-
-To run the HTTP API, run:
-
-    $ docker run \
-        --init \
-        -e RUST_LOG=info \
-        -p 8777:8777 \
-        -v $HOME/.radicle:/app/radicle radicle-services/http-api \
-        --tls-cert /app/radicle/fullchain.pem \
-        --tls-key /app/radicle/privkey.pem
-
-Make sure your TLS certificate files can be found under `$HOME/.radicle`. If you
-are using TLS termination, simply omit the `--tls-*` arguments.
-
-You may also want to detach the process (`-d`) and run with a TTY in interactive
-mode (`-it`).
+1. [Install docker-compose](https://docs.docker.com/compose/install/).
+1. `export DOCKER_TAG=$(git rev-parse HEAD)`
+1. `export SEED_USER_ID=$(id -u radicle-seed)` (not needed for building and pushing images, only for running)
+1. `docker-compose -f docker-compose.yml -f docker-compose.gcp.yml build`
+1. `docker-compose -f docker-compose.yml -f docker-compose.gcp.yml push`
```

Comment:
I'd rather we leave the readme as-is and later add a section on docker-compose, as some people may want to build it themselves.

Reply:
OK, reverted.